src/Tribe/Query.php

```diff
@@ -81,7 +81,7 @@ protected function filter_by_ticketed_status( WP_Query $query, bool $has_tickets
 				 * in the `wp_postmeta.meta_value` column for a fast query.
 				 */
 				$query = "SELECT p.ID FROM $wpdb->posts p
-				 JOIN $wpdb->postmeta pm ON ( $meta_keys_in ) AND pm.meta_value = p.ID
+				 JOIN $wpdb->postmeta pm ON ( $meta_keys_in ) AND pm.meta_value = CONCAT( p.ID, '' )
```

Comment:
If the purpose is to make sure the `p.ID` value will be passed to the query in string format instead of integer, why not use a `%s` placeholder and prepare the query?